homeassistant/components/cover/zwave.py

```diff
@@ -81,6 +66,8 @@ def update_properties(self):
         self._close_id = self.get_value(
             class_id=zwave.const.COMMAND_CLASS_SWITCH_MULTILEVEL,
             label=['Close', 'Down'], member='value_id')
+        if self._workaround == workaround.WORKAROUND_REVERSE_OPEN_CLOSE:
```

Comment:
I am pretty sure this should not be implemented. From helping out people in the past, this means you have reversed the wiring.